terraform/README.md

```diff
@@ -1,3 +1,26 @@
 # SIPMediaDeploy - Terraform
 
 A [Terraform](https://www.terraform.io/docs) project to deploy the SIP Media Gateway stack.
+
+
+## Installation
+
+Install Terraform using the [docs](https://learn.hashicorp.com/tutorials/terraform/install-cli), depending on your OS.
```

Comment:
Maybe you should use docker to run ansible and terraform so that people don't have to install anything on their machines?  This will also ensure that tey run it with the same version as what you planned.
It's done here for example: https://github.com/openfun/joanie/blob/main/docs/media.md#setup-a-shared-terraform-state

Reply:
Good idea! We will think about it and maybe implement it in a future PR